EIPS/eip-TBD.md

```diff
@@ -1,85 +1,57 @@
-
-  TODO: Remove this comment before submitting
--->
-
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
 
-For each `tokenId` we introduce a set of addresses whose members MAY transfer the token to themselves without payment by calling `TBD()`, which we will refer to as *take-back*.
+For each `tokenId` we introduce a set of addresses whose members MAY transfer the token to themselves without payment by calling `TBD()`, which we will refer to as a *take-back*.
 The set MUST be initially empty.
-For every [ERC-721](./eip-721.md) transfer, *except* for initial mint, the set of the corresponding `tokenId` MUST be appended with the previous owner (i.e. the transfer's `from` address).
+For every ERC-721 transfer, *except* for initial mint, the set of the corresponding `tokenId` MUST be appended with the previous owner (i.e. the transfer's `from` address).
 
-Royalty payment by the token owner MUST be accepted by the token contract via a dedicated function, `TBD()`.
+Royalty payment by the new token owner MUST be accepted by the token contract via a dedicated function, `TBD()`.
 Upon receipt of royalties, the take-back set MUST be emptied (i.e. transfer permissions collapse back to standard ERC-721 rules).
-After each token transfer, there SHOULD be a grace period during which the token is frozen and take-backs are disabled to allow the new owner to decide whether to pay the royalty or allow the take-back set to grow.
+After each token transfer, there SHOULD be a grace period during which the token is frozen and take-backs are disabled to allow the new owner to decide whether to either pay the royalty or allow the take-back set to grow.
 
 In addition to clearing the take-back set, receipt of royalties MUST begin a temporary window during which *any* address MAY purchase the token for a price that is a function of the royalty received (which can be viewed as the inverse of the `royaltyInfo()` function of ERC-2981).
 This process is referred to as *auto-listing* and is not limited to addresses in the take-back set.
-The token MUST NOT be transferrable for the duration of the *auto-listing*.
+The token MUST NOT be transferrable for the duration of the auto-listing.
 Implementations MAY specify the duration of the auto-listing window at their discretion but it MUST be finite and SHOULD be long enough to allow arbitrage opportunities to be detected and, after expiration, transfer permissions MUST collapse back to standard ERC-721 rules.
 
+In the event of a sale due to an auto-listing, (a) the corresponding royalties MUST be reimbursed, (b) the auto-listing price MUST be transferred to the token owner (i.e. the same address receives both the royalties and the sale price), and (c) the token MUST be transferred to the address paying the price.
```

Comment:
rearranging the points to make it clearer who the previous owner is
```suggestion
In the event of a sale due to an auto-listing, the token MUST be transferred to the address paying the auto-listing price, the price MUST be transferred to the previous token owner and the previously paid royalties MUST be reimbursed (i.e. the same address receives both the royalties and the sale price).
```

Reply:
Done. Not committing the suggestion directly as I've made some other changes related to the earlier comments.